web-server/helpers.js

```diff
@@ -133,7 +133,7 @@ module.exports = app => {
         })
     }
 
-    response.renderMarkdown = function(markdown){
+    response.renderMarkdown = function(markdown, viewPage='markdown', extraArgs){
```

Comment:
i dont want to do this this way. please restore these helpers and see my comment in the routes file

Reply:
add this helper `response.renderMarkdownFileToHTML` that does most of what `renderMarkdownFile` does but yields the HTML. then update `renderMarkdownFile` to use the `renderMarkdownFileToHTML`